tony-history-server/build.gradle

```diff
@@ -0,0 +1,47 @@
+plugins {
+    id 'play'
+    id 'idea'
+}
+
+def playVersion = "2.6.20"
+def hadoopVersion = "3.1.1"
+def scalaVersion = System.getProperty("scala.binary.version", "2.12")
+
+model {
+    components {
+        play {
+            platform play: playVersion, scala: scalaVersion, java: '1.8'
+            injectedRoutesGenerator = true
+
+            sources {
+                twirlTemplates {
+                    defaultImports = TwirlImports.JAVA
+                }
+            }
+        }
+    }
+}
+
+dependencies {
+    play "com.typesafe.play:play-guice_$scalaVersion:$playVersion"
+    play "com.typesafe.play:play-logback_$scalaVersion:$playVersion"
+    play "org.apache.hadoop:hadoop-hdfs-client:$hadoopVersion"
+    play "org.apache.hadoop:hadoop-common:$hadoopVersion"
+    play "org.apache.hadoop:hadoop-yarn-api:$hadoopVersion"
+
+    playTest "org.assertj:assertj-core:3.6.2"
+    playTest "org.awaitility:awaitility:2.0.0"
+}
+
+repositories {
+    jcenter()
+    maven {
+        name "lightbend-maven-release"
+        url "https://repo.lightbend.com/lightbend/maven-releases"
+    }
+    ivy {
+        name "lightbend-ivy-release"
+        url "https://repo.lightbend.com/lightbend/ivy-releases"
```

Comment:
Are all the dependencies required to build present in our internal Artifactory? If not, we should request to import them.

Reply:
I'm not sure. All I know from gradle is that you have to include them. https://docs.gradle.org/current/userguide/play_plugin.html#sec:play_usage

Reply:
Looks like we have the Scala 2.11 versions of play-guice and play-logback internally, but not the 2.12 ones. Please use Scala 2.11 (see my other comment).